Prose/ProseLib/Sources/ProseCoreTCA/Types/JID.swift

```diff
@@ -19,6 +19,12 @@ public extension JID {
     }
 }
 
+extension JID: Comparable {
+    public static func < (lhs: Self, rhs: Self) -> Bool {
+        lhs.rawValue < rhs.rawValue
+    }
+}
```

Comment:
Why do you need this? Is it for user-facing features? If it's the case, we should use a comparator like `String.Comparator(options: [.caseInsensitive, .diacriticInsensitive, .numeric]))` (changing the options as needed)